README.md

```diff
@@ -1,4 +1,4 @@
-# ![nf-core/epitopeprediction](docs/images/nf-core-epitopeprediction_logo_light.png#gh-light-mode-only) ![nf-core/epitopeprediction](docs/images/nf-core-epitopeprediction_logo_dark.png#gh-dark-mode-only)
+# ![nf-core/epitopeprediction](docs/images/nf-core/epitopeprediction_logo_light.png#gh-light-mode-only) ![nf-core/epitopeprediction](docs/images/nf-core/epitopeprediction_logo_dark.png#gh-dark-mode-only)
```

Comment:
That breaks the logo, thats a template bug imho, had this with all pipelines that were updated:
```suggestion
# ![nf-core/epitopeprediction](docs/images/nf-core-epitopeprediction_logo_light.png#gh-light-mode-only) ![nf-core/epitopeprediction](docs/images/nf-core-epitopeprediction_logo_dark.png#gh-dark-mode-only)
```